### PR TITLE
Add "One Process Per Container" image

### PR DESCRIPTION
**Pull request by shusugmt**

### CODE REVIEW

**Comment by warrenvw** [comment]
This is an amazing PR. It's been fun taking it for a spin, and I look forward to the cool things it'll enable moving forward!

File: examples/kubernetes-1ppc/configmaps.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: st2
+data:
+  ST2_USER: admin
+  ST2_PASSWORD: changeme
```
Comment (shusugmt) [inline examples/kubernetes-1ppc/configmaps.yml]: I'm not sure. This manifest is just an example so I guess it's fine with `changeme` but of course I can put some random chars in there. If one want to deploy st2 on k8s, they should provide their own `configmap` or even they should use k8s `secret` for sensitive information like passwords.

File: images/stackstorm-1ppc/nginx_st2.conf [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  listen       *:443 ssl;
+
+  ssl on;
+
+  ssl_certificate           /etc/ssl/st2/st2.crt;
+  ssl_certificate_key       /etc/ssl/st2/st2.key;
+  ssl_session_cache         shared:SSL:10m;
+  ssl_session_timeout       5m;
+  ssl_protocols             TLSv1 TLSv1.1 TLSv1.2;
+  ssl_ciphers               EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH:ECDHE-RSA-AES128-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA128:DHE-RSA-AES128-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES128-GCM-SHA128:ECDHE-RSA-AES128-SHA384:ECDHE-RSA-AES128-SHA128:ECDHE-RSA-AES128-SHA:ECDHE-RSA-AES128-SHA:DHE-RSA-AES128-SHA128:DHE-RSA-AES128-SHA128:DHE-RSA-AES128-SHA:DHE-RSA-AES128-SHA:AES128-GCM-SHA384:AES128-GCM-SHA128:AES128-SHA128:AES128-SHA128:AES128-SHA:AES128-SHA:HIGH:!aNULL:!eNULL:!EXPORT:!DES:!MD5:!PSK:!RC4;
```
Comment (shusugmt) [inline images/stackstorm-1ppc/nginx_st2.conf]: This nginx_st2.conf is based on `/etc/nginx/conf.d/st2.conf` in `stackstorm:stackstorm` image, and the modification is made as minimal as possible. I haven't touch those parameters, so if it's necessary then it should be fixed in upstream package I guess?

File: images/stackstorm-1ppc/entrypoint-1ppc.sh [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+crudini --set ${ST2_CONF} api allow_origin '*'
+
+
+case "$ST2_SERVICE" in
+  "nop" )
+    exec tail -f /dev/null
+    ;;
+  "st2api" )
+    DAEMON_ARGS="-k eventlet -b 0.0.0.0:9101 --workers 1 --threads 1 --graceful-timeout 10 --timeout 30"
+    exec /opt/stackstorm/st2/bin/gunicorn_pecan /opt/stackstorm/st2/lib/python2.7/site-packages/st2api/gunicorn_config.py $DAEMON_ARGS
```
Comment (shusugmt) [inline images/stackstorm-1ppc/entrypoint-1ppc.sh]: These also came from original scripts in `/etc/init` like https://github.com/StackStorm/st2-packages/blob/v2.2/packages/st2/debian/st2.st2api.upstart and tried to use the exact same line (I made some necessary changes, though)

I just want to keep that way to make it reproducible for future version up.

File: examples/kubernetes-1ppc/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+Tested environment:
+
+- Mac
+    - minikube version: v0.19.1
+    - Kubernetes v1.6.4
+
+```
+# Start minikube cluster
+# Note: Allow assigning 443 for NodePort service, in order to access st2web
+minikube start --driver=xhyve --extra-config=apiserver.ServiceNodePortRange=443-32767
```
Comment (shusugmt) [inline examples/kubernetes-1ppc/README.md]: Sorry for the typo and confusion. Will update.

**Comment by shusugmt** [comment]
I haven't come up with a good name for a directory to put example k8s manifest files yet... For now I'll put them under `runtime`

File: images/stackstorm-1ppc/nginx_st2.conf (hunk 2 above)
Comment (warrenvw) [inline images/stackstorm-1ppc/nginx_st2.conf]: Yes, I agree.

**Pull request merged**